src/TimeSteppers/split_hydrostatic_runge_kutta_3.jl

```diff
@@ -48,8 +49,8 @@ The state at the first substep is taken to be the one that corresponds to the ``
 function SplitRungeKutta3TimeStepper(grid, tracers;
                                      implicit_solver::TI = nothing,
                                      Gⁿ::TG = TendencyFields(grid, tracers),
-                                     G⁻::TE = TendencyFields(grid, tracers),
-                                     Ψ⁻::PF = TendencyFields(grid, tracers)) where {TI, TG, TE, PF}
+                                     G⁻::TE = nothing,
+                                     prognostic_fields = TendencyFields(grid, tracers)) where {TI, TG, TE}
```

Comment:
This default doesn't make any sense right? Maybe this should just be a positional required argument